app/konnect/org-management/okta-idp.md

```diff
@@ -3,37 +3,20 @@ title: Set Up SSO with Okta
 badge: enterprise
 ---
 
-As an alternative to {{site.konnect_saas}}’s native authentication, you can set
-up single sign-on (SSO) access to {{site.konnect_short_name}} through
-[Okta](https://developer.okta.com/docs/guides/) with
-[OpenID Connect](https://developer.okta.com/docs/concepts/oauth-openid/#openid-connect).
-This authentication method allows your users to log in to {{site.konnect_saas}}
-using their Okta credentials, without needing a separate login.
 
+As an alternative to {{site.konnect_saas}}’s native authentication, you can set up single sign-on (SSO) access to {{site.konnect_short_name}} through Okta using OpenID Connect or SAML. These authentication methods allow your users to log in to {{site.konnect_saas}} using their Okta credentials without needing a separate login.
 
-## Prerequisites and overview of steps
-
-To set up Okta single sign-on (SSO) for {{site.konnect_short_name}}, you need
-access to an Okta admin account and a
-[{{site.konnect_short_name}} admin account](/konnect/org-management/teams-and-roles),
-which you will access concurrently.
 
-Optionally, if you want to use team mappings, you must configure Okta to include group claims in the ID token. 
+You cannot mix authenticators in {{site.konnect_saas}}. With Okta authentication enabled, all non-admin {{site.konnect_short_name}} users must log in through Okta. Only the {{site.konnect_short_name}} org owner can continue to log in with {{site.konnect_short_name}}'s native authentication.
```

Comment:
`must` is too strong of a word. It is possible to leave "built-in" authentication enable and allow users to choose between SSO and other authenticators.